tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -23,6 +23,8 @@ namespace spirv {
 
 namespace {
 
+const char *glslExtInstSetName() { return "GLSL.std.450"; }
```

Comment:
For example, this one can be declared as `static const char* const`. Should be safe enough. :)